2014/12/03 05:14:26 [error] [exception.CHttpException.404] exception 'CHttpException' in C:\newOpenServer\domains\ybugs.local\protected\controllers\IssuesController.php:107
Stack trace:
#0 [internal function]: IssuesController->actionAdd('')
#1 C:\newOpenServer\domains\ybugs.local\framework\web\actions\CAction.php(108): ReflectionMethod->invokeArgs(Object(IssuesController), Array)
#2 C:\newOpenServer\domains\ybugs.local\framework\web\actions\CInlineAction.php(47): CAction->runWithParamsInternal(Object(IssuesController), Object(ReflectionMethod), Array)
#3 C:\newOpenServer\domains\ybugs.local\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 C:\newOpenServer\domains\ybugs.local\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 C:\newOpenServer\domains\ybugs.local\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 C:\newOpenServer\domains\ybugs.local\framework\web\CWebApplication.php(282): CController->run('add')
#7 C:\newOpenServer\domains\ybugs.local\framework\web\CWebApplication.php(141): CWebApplication->runController('issues/add/id')
#8 C:\newOpenServer\domains\ybugs.local\framework\base\CApplication.php(180): CWebApplication->processRequest()
#9 C:\newOpenServer\domains\ybugs.local\index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/issues/add/id/
---
2014/12/03 05:14:32 [error] [exception.CHttpException.404] exception 'CHttpException' in C:\newOpenServer\domains\ybugs.local\protected\controllers\IssuesController.php:107
Stack trace:
#0 [internal function]: IssuesController->actionAdd(NULL)
#1 C:\newOpenServer\domains\ybugs.local\framework\web\actions\CAction.php(108): ReflectionMethod->invokeArgs(Object(IssuesController), Array)
#2 C:\newOpenServer\domains\ybugs.local\framework\web\actions\CInlineAction.php(47): CAction->runWithParamsInternal(Object(IssuesController), Object(ReflectionMethod), Array)
#3 C:\newOpenServer\domains\ybugs.local\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 C:\newOpenServer\domains\ybugs.local\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 C:\newOpenServer\domains\ybugs.local\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 C:\newOpenServer\domains\ybugs.local\framework\web\CWebApplication.php(282): CController->run('add')
#7 C:\newOpenServer\domains\ybugs.local\framework\web\CWebApplication.php(141): CWebApplication->runController('issues/add')
#8 C:\newOpenServer\domains\ybugs.local\framework\base\CApplication.php(180): CWebApplication->processRequest()
#9 C:\newOpenServer\domains\ybugs.local\index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/issues/add/
---
2014/12/03 22:08:50 [error] [exception.CException] exception 'CException' with message 'ExtUserRoles and its behaviors do not have a method or closure named "getUseRole".' in C:\newOpenServer\domains\ybugs.local\framework\base\CComponent.php:266
Stack trace:
#0 C:\newOpenServer\domains\ybugs.local\framework\db\ar\CActiveRecord.php(227): CComponent->__call('getUseRole', Array)
#1 C:\newOpenServer\domains\ybugs.local\protected\controllers\UsersController.php(110): CActiveRecord->__call('getUseRole', Array)
#2 C:\newOpenServer\domains\ybugs.local\protected\controllers\UsersController.php(110): ExtUserRoles->getUseRole('1')
#3 [internal function]: UsersController->actionAddPrj('1')
#4 C:\newOpenServer\domains\ybugs.local\framework\web\actions\CAction.php(108): ReflectionMethod->invokeArgs(Object(UsersController), Array)
#5 C:\newOpenServer\domains\ybugs.local\framework\web\actions\CInlineAction.php(47): CAction->runWithParamsInternal(Object(UsersController), Object(ReflectionMethod), Array)
#6 C:\newOpenServer\domains\ybugs.local\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#7 C:\newOpenServer\domains\ybugs.local\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#8 C:\newOpenServer\domains\ybugs.local\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#9 C:\newOpenServer\domains\ybugs.local\framework\web\CWebApplication.php(282): CController->run('addprj')
#10 C:\newOpenServer\domains\ybugs.local\framework\web\CWebApplication.php(141): CWebApplication->runController('users/addprj')
#11 C:\newOpenServer\domains\ybugs.local\framework\base\CApplication.php(180): CWebApplication->processRequest()
#12 C:\newOpenServer\domains\ybugs.local\index.php(13): CApplication->run()
#13 {main}
REQUEST_URI=/en/users/addprj/1
HTTP_REFERER=http://ybugs.local/en/users/list
---
2014/12/03 22:13:31 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "user/list".' in C:\newOpenServer\domains\ybugs.local\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\newOpenServer\domains\ybugs.local\framework\web\CWebApplication.php(141): CWebApplication->runController('user/list')
#1 C:\newOpenServer\domains\ybugs.local\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 C:\newOpenServer\domains\ybugs.local\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/en/user/list
HTTP_REFERER=http://ybugs.local/en/users/addprj/1
---
2014/12/03 21:20:32 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "project/del".' in /Applications/AMPPS/www/ybugs.local/framework/web/CWebApplication.php:286
Stack trace:
#0 /Applications/AMPPS/www/ybugs.local/framework/web/CWebApplication.php(141): CWebApplication->runController('project/del')
#1 /Applications/AMPPS/www/ybugs.local/framework/base/CApplication.php(180): CWebApplication->processRequest()
#2 /Applications/AMPPS/www/ybugs.local/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/en/project/del
HTTP_REFERER=http://ybugs.local/en/main/index
---
2014/12/03 21:20:58 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "project/del".' in /Applications/AMPPS/www/ybugs.local/framework/web/CWebApplication.php:286
Stack trace:
#0 /Applications/AMPPS/www/ybugs.local/framework/web/CWebApplication.php(141): CWebApplication->runController('project/del')
#1 /Applications/AMPPS/www/ybugs.local/framework/base/CApplication.php(180): CWebApplication->processRequest()
#2 /Applications/AMPPS/www/ybugs.local/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/en/project/del
HTTP_REFERER=http://ybugs.local/en/main/index
---
2014/12/03 21:21:35 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "project/del".' in /Applications/AMPPS/www/ybugs.local/framework/web/CWebApplication.php:286
Stack trace:
#0 /Applications/AMPPS/www/ybugs.local/framework/web/CWebApplication.php(141): CWebApplication->runController('project/del')
#1 /Applications/AMPPS/www/ybugs.local/framework/base/CApplication.php(180): CWebApplication->processRequest()
#2 /Applications/AMPPS/www/ybugs.local/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/en/project/del
HTTP_REFERER=http://ybugs.local/en/main/index
---
2014/12/03 21:21:46 [error] [exception.CHttpException.400] exception 'CHttpException' with message 'Your request is invalid.' in /Applications/AMPPS/www/ybugs.local/framework/web/CController.php:336
Stack trace:
#0 /Applications/AMPPS/www/ybugs.local/framework/web/CController.php(309): CController->invalidActionParams(Object(CInlineAction))
#1 /Applications/AMPPS/www/ybugs.local/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#2 /Applications/AMPPS/www/ybugs.local/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#3 /Applications/AMPPS/www/ybugs.local/framework/web/CWebApplication.php(282): CController->run('del')
#4 /Applications/AMPPS/www/ybugs.local/framework/web/CWebApplication.php(141): CWebApplication->runController('projects/del')
#5 /Applications/AMPPS/www/ybugs.local/framework/base/CApplication.php(180): CWebApplication->processRequest()
#6 /Applications/AMPPS/www/ybugs.local/index.php(13): CApplication->run()
#7 {main}
REQUEST_URI=/en/projects/del
HTTP_REFERER=http://ybugs.local/en/main/index
---
2014/12/03 21:42:41 [error] [php] strpos() expects parameter 1 to be string, object given (/Applications/AMPPS/www/ybugs.local/framework/web/CHttpRequest.php:801)
Stack trace:
#0 /Applications/AMPPS/www/ybugs.local/protected/controllers/ProjectsController.php(99): ProjectsController->redirect()
#1 unknown(0): ProjectsController->actionDel()
#2 /Applications/AMPPS/www/ybugs.local/framework/web/actions/CAction.php(108): ReflectionMethod->invokeArgs()
#3 /Applications/AMPPS/www/ybugs.local/framework/web/actions/CInlineAction.php(47): CInlineAction->runWithParamsInternal()
#4 /Applications/AMPPS/www/ybugs.local/framework/web/CController.php(308): CInlineAction->runWithParams()
#5 /Applications/AMPPS/www/ybugs.local/framework/web/CController.php(286): ProjectsController->runAction()
#6 /Applications/AMPPS/www/ybugs.local/framework/web/CController.php(265): ProjectsController->runActionWithFilters()
#7 /Applications/AMPPS/www/ybugs.local/framework/web/CWebApplication.php(282): ProjectsController->run()
#8 /Applications/AMPPS/www/ybugs.local/framework/web/CWebApplication.php(141): CWebApplication->runController()
#9 /Applications/AMPPS/www/ybugs.local/framework/base/CApplication.php(180): CWebApplication->processRequest()
#10 /Applications/AMPPS/www/ybugs.local/index.php(13): CWebApplication->run()
REQUEST_URI=/en/projects/del/10
in /Applications/AMPPS/www/ybugs.local/protected/controllers/ProjectsController.php (99)
in /Applications/AMPPS/www/ybugs.local/index.php (13)
2014/12/03 21:43:14 [error] [exception.CHttpException.400] exception 'CHttpException' with message 'Your request is invalid.' in /Applications/AMPPS/www/ybugs.local/framework/web/CController.php:336
Stack trace:
#0 /Applications/AMPPS/www/ybugs.local/framework/web/CController.php(309): CController->invalidActionParams(Object(CInlineAction))
#1 /Applications/AMPPS/www/ybugs.local/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#2 /Applications/AMPPS/www/ybugs.local/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#3 /Applications/AMPPS/www/ybugs.local/framework/web/CWebApplication.php(282): CController->run('del')
#4 /Applications/AMPPS/www/ybugs.local/framework/web/CWebApplication.php(141): CWebApplication->runController('projects/del')
#5 /Applications/AMPPS/www/ybugs.local/framework/base/CApplication.php(180): CWebApplication->processRequest()
#6 /Applications/AMPPS/www/ybugs.local/index.php(13): CApplication->run()
#7 {main}
REQUEST_URI=/en/projects/del/list
HTTP_REFERER=http://ybugs.local/en/main/index
---
2014/12/04 01:38:06 [error] [exception.CHttpException.400] exception 'CHttpException' with message 'Your request is invalid.' in /Applications/AMPPS/www/ybugs.local/framework/web/CController.php:336
Stack trace:
#0 /Applications/AMPPS/www/ybugs.local/framework/web/CController.php(309): CController->invalidActionParams(Object(CInlineAction))
#1 /Applications/AMPPS/www/ybugs.local/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#2 /Applications/AMPPS/www/ybugs.local/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#3 /Applications/AMPPS/www/ybugs.local/framework/web/CWebApplication.php(282): CController->run('delres')
#4 /Applications/AMPPS/www/ybugs.local/framework/web/CWebApplication.php(141): CWebApplication->runController('issues/delres/1...')
#5 /Applications/AMPPS/www/ybugs.local/framework/base/CApplication.php(180): CWebApplication->processRequest()
#6 /Applications/AMPPS/www/ybugs.local/index.php(13): CApplication->run()
#7 {main}
REQUEST_URI=/issues/delres/15
HTTP_REFERER=http://ybugs.local/en/issues/getissue/23
---
2014/12/04 01:38:08 [error] [exception.CHttpException.400] exception 'CHttpException' with message 'Your request is invalid.' in /Applications/AMPPS/www/ybugs.local/framework/web/CController.php:336
Stack trace:
#0 /Applications/AMPPS/www/ybugs.local/framework/web/CController.php(309): CController->invalidActionParams(Object(CInlineAction))
#1 /Applications/AMPPS/www/ybugs.local/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#2 /Applications/AMPPS/www/ybugs.local/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#3 /Applications/AMPPS/www/ybugs.local/framework/web/CWebApplication.php(282): CController->run('delres')
#4 /Applications/AMPPS/www/ybugs.local/framework/web/CWebApplication.php(141): CWebApplication->runController('issues/delres/1...')
#5 /Applications/AMPPS/www/ybugs.local/framework/base/CApplication.php(180): CWebApplication->processRequest()
#6 /Applications/AMPPS/www/ybugs.local/index.php(13): CApplication->run()
#7 {main}
REQUEST_URI=/issues/delres/15
HTTP_REFERER=http://ybugs.local/en/issues/getissue/23
---
2014/12/04 01:38:08 [error] [exception.CHttpException.400] exception 'CHttpException' with message 'Your request is invalid.' in /Applications/AMPPS/www/ybugs.local/framework/web/CController.php:336
Stack trace:
#0 /Applications/AMPPS/www/ybugs.local/framework/web/CController.php(309): CController->invalidActionParams(Object(CInlineAction))
#1 /Applications/AMPPS/www/ybugs.local/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#2 /Applications/AMPPS/www/ybugs.local/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#3 /Applications/AMPPS/www/ybugs.local/framework/web/CWebApplication.php(282): CController->run('delres')
#4 /Applications/AMPPS/www/ybugs.local/framework/web/CWebApplication.php(141): CWebApplication->runController('issues/delres/1...')
#5 /Applications/AMPPS/www/ybugs.local/framework/base/CApplication.php(180): CWebApplication->processRequest()
#6 /Applications/AMPPS/www/ybugs.local/index.php(13): CApplication->run()
#7 {main}
REQUEST_URI=/issues/delres/15
HTTP_REFERER=http://ybugs.local/en/issues/getissue/23
---
2014/12/04 01:38:08 [error] [exception.CHttpException.400] exception 'CHttpException' with message 'Your request is invalid.' in /Applications/AMPPS/www/ybugs.local/framework/web/CController.php:336
Stack trace:
#0 /Applications/AMPPS/www/ybugs.local/framework/web/CController.php(309): CController->invalidActionParams(Object(CInlineAction))
#1 /Applications/AMPPS/www/ybugs.local/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#2 /Applications/AMPPS/www/ybugs.local/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#3 /Applications/AMPPS/www/ybugs.local/framework/web/CWebApplication.php(282): CController->run('delres')
#4 /Applications/AMPPS/www/ybugs.local/framework/web/CWebApplication.php(141): CWebApplication->runController('issues/delres/1...')
#5 /Applications/AMPPS/www/ybugs.local/framework/base/CApplication.php(180): CWebApplication->processRequest()
#6 /Applications/AMPPS/www/ybugs.local/index.php(13): CApplication->run()
#7 {main}
REQUEST_URI=/issues/delres/15
HTTP_REFERER=http://ybugs.local/en/issues/getissue/23
---
2015/03/13 18:22:34 [error] [system.db.CDbCommand] Error in preparing SQL: SELECT t1.id,t2.text,t1.translation, t2.id AS message_id
            FROM message_trl t1
            JOIN message t2 ON t1.message_id = t2.id
            JOIN languages t3 ON t1.language_id = t3.id
          WHERE t3.prefix = :prefix ORDER BY t1.message_id DESC
in C:\OpenServer\domains\ybugs.local\protected\models\ext\ExtMessage.php (55)
in C:\OpenServer\domains\ybugs.local\protected\controllers\LanguagesController.php (246)
in C:\OpenServer\domains\ybugs.local\protected\controllers\LanguagesController.php (276)
2015/03/13 18:22:34 [error] [system.db.CDbCommand] CDbCommand::fetchAll() failed: CDbCommand failed to prepare the SQL statement: SQLSTATE[HY000]: General error: 1 no such column: t1.id. The SQL statement executed was: SELECT t1.id,t2.text,t1.translation, t2.id AS message_id
            FROM message_trl t1
            JOIN message t2 ON t1.message_id = t2.id
            JOIN languages t3 ON t1.language_id = t3.id
          WHERE t3.prefix = :prefix ORDER BY t1.message_id DESC.
in C:\OpenServer\domains\ybugs.local\protected\models\ext\ExtMessage.php (55)
in C:\OpenServer\domains\ybugs.local\protected\controllers\LanguagesController.php (246)
in C:\OpenServer\domains\ybugs.local\protected\controllers\LanguagesController.php (276)
2015/03/13 18:22:34 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand failed to execute the SQL statement: CDbCommand failed to prepare the SQL statement: SQLSTATE[HY000]: General error: 1 no such column: t1.id. The SQL statement executed was: SELECT t1.id,t2.text,t1.translation, t2.id AS message_id
            FROM message_trl t1
            JOIN message t2 ON t1.message_id = t2.id
            JOIN languages t3 ON t1.language_id = t3.id
          WHERE t3.prefix = :prefix ORDER BY t1.message_id DESC' in C:\OpenServer\domains\ybugs.local\framework\db\CDbCommand.php:543
Stack trace:
#0 C:\OpenServer\domains\ybugs.local\framework\db\CDbCommand.php(396): CDbCommand->queryInternal('fetchAll', Array, Array)
#1 C:\OpenServer\domains\ybugs.local\protected\models\ext\ExtMessage.php(55): CDbCommand->queryAll(true, Array)
#2 C:\OpenServer\domains\ybugs.local\protected\controllers\LanguagesController.php(246): ExtMessage->getMessagesArr('en', Array)
#3 C:\OpenServer\domains\ybugs.local\protected\controllers\LanguagesController.php(276): LanguagesController->actionListMes()
#4 [internal function]: LanguagesController->actionMessages(NULL, NULL)
#5 C:\OpenServer\domains\ybugs.local\framework\web\actions\CAction.php(108): ReflectionMethod->invokeArgs(Object(LanguagesController), Array)
#6 C:\OpenServer\domains\ybugs.local\framework\web\actions\CInlineAction.php(47): CAction->runWithParamsInternal(Object(LanguagesController), Object(ReflectionMethod), Array)
#7 C:\OpenServer\domains\ybugs.local\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#8 C:\OpenServer\domains\ybugs.local\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#9 C:\OpenServer\domains\ybugs.local\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#10 C:\OpenServer\domains\ybugs.local\framework\web\CWebApplication.php(282): CController->run('messages')
#11 C:\OpenServer\domains\ybugs.local\framework\web\CWebApplication.php(141): CWebApplication->runController('languages/messa...')
#12 C:\OpenServer\domains\ybugs.local\framework\base\CApplication.php(180): CWebApplication->processRequest()
#13 C:\OpenServer\domains\ybugs.local\index.php(13): CApplication->run()
#14 {main}
REQUEST_URI=/en/languages/messages
HTTP_REFERER=http://ybugs.local/en/languages
---
2015/03/13 18:23:44 [error] [system.db.CDbCommand] Error in preparing SQL: SELECT t1.id,t2.text,t1.translation, t2.id AS message_id
            FROM message_trl t1
            JOIN message t2 ON t1.message_id = t2.id
            JOIN languages t3 ON t1.language_id = t3.id
          WHERE t3.prefix = :prefix ORDER BY t1.message_id DESC
in C:\OpenServer\domains\ybugs.local\protected\models\ext\ExtMessage.php (55)
in C:\OpenServer\domains\ybugs.local\protected\controllers\LanguagesController.php (246)
in C:\OpenServer\domains\ybugs.local\protected\controllers\LanguagesController.php (276)
2015/03/13 18:23:44 [error] [system.db.CDbCommand] CDbCommand::fetchAll() failed: CDbCommand failed to prepare the SQL statement: SQLSTATE[HY000]: General error: 1 no such column: t1.id. The SQL statement executed was: SELECT t1.id,t2.text,t1.translation, t2.id AS message_id
            FROM message_trl t1
            JOIN message t2 ON t1.message_id = t2.id
            JOIN languages t3 ON t1.language_id = t3.id
          WHERE t3.prefix = :prefix ORDER BY t1.message_id DESC.
in C:\OpenServer\domains\ybugs.local\protected\models\ext\ExtMessage.php (55)
in C:\OpenServer\domains\ybugs.local\protected\controllers\LanguagesController.php (246)
in C:\OpenServer\domains\ybugs.local\protected\controllers\LanguagesController.php (276)
2015/03/13 18:23:44 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand failed to execute the SQL statement: CDbCommand failed to prepare the SQL statement: SQLSTATE[HY000]: General error: 1 no such column: t1.id. The SQL statement executed was: SELECT t1.id,t2.text,t1.translation, t2.id AS message_id
            FROM message_trl t1
            JOIN message t2 ON t1.message_id = t2.id
            JOIN languages t3 ON t1.language_id = t3.id
          WHERE t3.prefix = :prefix ORDER BY t1.message_id DESC' in C:\OpenServer\domains\ybugs.local\framework\db\CDbCommand.php:543
Stack trace:
#0 C:\OpenServer\domains\ybugs.local\framework\db\CDbCommand.php(396): CDbCommand->queryInternal('fetchAll', Array, Array)
#1 C:\OpenServer\domains\ybugs.local\protected\models\ext\ExtMessage.php(55): CDbCommand->queryAll(true, Array)
#2 C:\OpenServer\domains\ybugs.local\protected\controllers\LanguagesController.php(246): ExtMessage->getMessagesArr('en', Array)
#3 C:\OpenServer\domains\ybugs.local\protected\controllers\LanguagesController.php(276): LanguagesController->actionListMes()
#4 [internal function]: LanguagesController->actionMessages(NULL, NULL)
#5 C:\OpenServer\domains\ybugs.local\framework\web\actions\CAction.php(108): ReflectionMethod->invokeArgs(Object(LanguagesController), Array)
#6 C:\OpenServer\domains\ybugs.local\framework\web\actions\CInlineAction.php(47): CAction->runWithParamsInternal(Object(LanguagesController), Object(ReflectionMethod), Array)
#7 C:\OpenServer\domains\ybugs.local\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#8 C:\OpenServer\domains\ybugs.local\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#9 C:\OpenServer\domains\ybugs.local\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#10 C:\OpenServer\domains\ybugs.local\framework\web\CWebApplication.php(282): CController->run('messages')
#11 C:\OpenServer\domains\ybugs.local\framework\web\CWebApplication.php(141): CWebApplication->runController('languages/messa...')
#12 C:\OpenServer\domains\ybugs.local\framework\base\CApplication.php(180): CWebApplication->processRequest()
#13 C:\OpenServer\domains\ybugs.local\index.php(13): CApplication->run()
#14 {main}
REQUEST_URI=/en/languages/messages
HTTP_REFERER=http://ybugs.local/en/languages
---
2015/03/13 18:24:08 [error] [system.db.CDbCommand] Error in preparing SQL: SELECT t1.id,t2.text,t1.translation, t2.id AS message_id
            FROM message_trl t1
            JOIN message t2 ON t1.message_id = t2.id
            JOIN languages t3 ON t1.language_id = t3.id
          WHERE t3.prefix = :prefix ORDER BY t1.message_id DESC
in C:\OpenServer\domains\ybugs.local\protected\models\ext\ExtMessage.php (55)
in C:\OpenServer\domains\ybugs.local\protected\controllers\LanguagesController.php (246)
in C:\OpenServer\domains\ybugs.local\protected\controllers\LanguagesController.php (276)
2015/03/13 18:24:08 [error] [system.db.CDbCommand] CDbCommand::fetchAll() failed: CDbCommand failed to prepare the SQL statement: SQLSTATE[HY000]: General error: 1 no such column: t1.id. The SQL statement executed was: SELECT t1.id,t2.text,t1.translation, t2.id AS message_id
            FROM message_trl t1
            JOIN message t2 ON t1.message_id = t2.id
            JOIN languages t3 ON t1.language_id = t3.id
          WHERE t3.prefix = :prefix ORDER BY t1.message_id DESC.
in C:\OpenServer\domains\ybugs.local\protected\models\ext\ExtMessage.php (55)
in C:\OpenServer\domains\ybugs.local\protected\controllers\LanguagesController.php (246)
in C:\OpenServer\domains\ybugs.local\protected\controllers\LanguagesController.php (276)
2015/03/13 18:24:08 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand failed to execute the SQL statement: CDbCommand failed to prepare the SQL statement: SQLSTATE[HY000]: General error: 1 no such column: t1.id. The SQL statement executed was: SELECT t1.id,t2.text,t1.translation, t2.id AS message_id
            FROM message_trl t1
            JOIN message t2 ON t1.message_id = t2.id
            JOIN languages t3 ON t1.language_id = t3.id
          WHERE t3.prefix = :prefix ORDER BY t1.message_id DESC' in C:\OpenServer\domains\ybugs.local\framework\db\CDbCommand.php:543
Stack trace:
#0 C:\OpenServer\domains\ybugs.local\framework\db\CDbCommand.php(396): CDbCommand->queryInternal('fetchAll', Array, Array)
#1 C:\OpenServer\domains\ybugs.local\protected\models\ext\ExtMessage.php(55): CDbCommand->queryAll(true, Array)
#2 C:\OpenServer\domains\ybugs.local\protected\controllers\LanguagesController.php(246): ExtMessage->getMessagesArr('en', Array)
#3 C:\OpenServer\domains\ybugs.local\protected\controllers\LanguagesController.php(276): LanguagesController->actionListMes()
#4 [internal function]: LanguagesController->actionMessages(NULL, NULL)
#5 C:\OpenServer\domains\ybugs.local\framework\web\actions\CAction.php(108): ReflectionMethod->invokeArgs(Object(LanguagesController), Array)
#6 C:\OpenServer\domains\ybugs.local\framework\web\actions\CInlineAction.php(47): CAction->runWithParamsInternal(Object(LanguagesController), Object(ReflectionMethod), Array)
#7 C:\OpenServer\domains\ybugs.local\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#8 C:\OpenServer\domains\ybugs.local\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#9 C:\OpenServer\domains\ybugs.local\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#10 C:\OpenServer\domains\ybugs.local\framework\web\CWebApplication.php(282): CController->run('messages')
#11 C:\OpenServer\domains\ybugs.local\framework\web\CWebApplication.php(141): CWebApplication->runController('languages/messa...')
#12 C:\OpenServer\domains\ybugs.local\framework\base\CApplication.php(180): CWebApplication->processRequest()
#13 C:\OpenServer\domains\ybugs.local\index.php(13): CApplication->run()
#14 {main}
REQUEST_URI=/en/languages/messages
HTTP_REFERER=http://ybugs.local/en/languages
---
2015/03/13 18:24:18 [error] [system.db.CDbCommand] Error in preparing SQL: SELECT t1.id,t2.text,t1.translation, t2.id AS message_id
            FROM message_trl t1
            JOIN message t2 ON t1.message_id = t2.id
            JOIN languages t3 ON t1.language_id = t3.id
          WHERE t3.prefix = :prefix ORDER BY t1.message_id DESC
in C:\OpenServer\domains\ybugs.local\protected\models\ext\ExtMessage.php (55)
in C:\OpenServer\domains\ybugs.local\protected\controllers\LanguagesController.php (246)
in C:\OpenServer\domains\ybugs.local\protected\controllers\LanguagesController.php (276)
2015/03/13 18:24:18 [error] [system.db.CDbCommand] CDbCommand::fetchAll() failed: CDbCommand failed to prepare the SQL statement: SQLSTATE[HY000]: General error: 1 no such column: t1.id. The SQL statement executed was: SELECT t1.id,t2.text,t1.translation, t2.id AS message_id
            FROM message_trl t1
            JOIN message t2 ON t1.message_id = t2.id
            JOIN languages t3 ON t1.language_id = t3.id
          WHERE t3.prefix = :prefix ORDER BY t1.message_id DESC.
in C:\OpenServer\domains\ybugs.local\protected\models\ext\ExtMessage.php (55)
in C:\OpenServer\domains\ybugs.local\protected\controllers\LanguagesController.php (246)
in C:\OpenServer\domains\ybugs.local\protected\controllers\LanguagesController.php (276)
2015/03/13 18:24:18 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand failed to execute the SQL statement: CDbCommand failed to prepare the SQL statement: SQLSTATE[HY000]: General error: 1 no such column: t1.id. The SQL statement executed was: SELECT t1.id,t2.text,t1.translation, t2.id AS message_id
            FROM message_trl t1
            JOIN message t2 ON t1.message_id = t2.id
            JOIN languages t3 ON t1.language_id = t3.id
          WHERE t3.prefix = :prefix ORDER BY t1.message_id DESC' in C:\OpenServer\domains\ybugs.local\framework\db\CDbCommand.php:543
Stack trace:
#0 C:\OpenServer\domains\ybugs.local\framework\db\CDbCommand.php(396): CDbCommand->queryInternal('fetchAll', Array, Array)
#1 C:\OpenServer\domains\ybugs.local\protected\models\ext\ExtMessage.php(55): CDbCommand->queryAll(true, Array)
#2 C:\OpenServer\domains\ybugs.local\protected\controllers\LanguagesController.php(246): ExtMessage->getMessagesArr('en', Array)
#3 C:\OpenServer\domains\ybugs.local\protected\controllers\LanguagesController.php(276): LanguagesController->actionListMes()
#4 [internal function]: LanguagesController->actionMessages(NULL, NULL)
#5 C:\OpenServer\domains\ybugs.local\framework\web\actions\CAction.php(108): ReflectionMethod->invokeArgs(Object(LanguagesController), Array)
#6 C:\OpenServer\domains\ybugs.local\framework\web\actions\CInlineAction.php(47): CAction->runWithParamsInternal(Object(LanguagesController), Object(ReflectionMethod), Array)
#7 C:\OpenServer\domains\ybugs.local\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#8 C:\OpenServer\domains\ybugs.local\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#9 C:\OpenServer\domains\ybugs.local\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#10 C:\OpenServer\domains\ybugs.local\framework\web\CWebApplication.php(282): CController->run('messages')
#11 C:\OpenServer\domains\ybugs.local\framework\web\CWebApplication.php(141): CWebApplication->runController('languages/messa...')
#12 C:\OpenServer\domains\ybugs.local\framework\base\CApplication.php(180): CWebApplication->processRequest()
#13 C:\OpenServer\domains\ybugs.local\index.php(13): CApplication->run()
#14 {main}
REQUEST_URI=/en/languages/messages
HTTP_REFERER=http://ybugs.local/en/languages
---
2015/03/13 18:26:53 [error] [system.db.CDbCommand] Error in preparing SQL: SELECT t1.id,t2.text,t1.translation, t2.id AS message_id
            FROM message_trl t1
            JOIN message t2 ON t1.message_id = t2.id
            JOIN languages t3 ON t1.language_id = t3.id
          WHERE t3.prefix = :prefix ORDER BY t1.message_id DESC
in C:\OpenServer\domains\ybugs.local\protected\models\ext\ExtMessage.php (55)
in C:\OpenServer\domains\ybugs.local\protected\controllers\LanguagesController.php (246)
in C:\OpenServer\domains\ybugs.local\protected\controllers\LanguagesController.php (276)
2015/03/13 18:26:53 [error] [system.db.CDbCommand] CDbCommand::fetchAll() failed: CDbCommand failed to prepare the SQL statement: SQLSTATE[HY000]: General error: 1 no such column: t1.id. The SQL statement executed was: SELECT t1.id,t2.text,t1.translation, t2.id AS message_id
            FROM message_trl t1
            JOIN message t2 ON t1.message_id = t2.id
            JOIN languages t3 ON t1.language_id = t3.id
          WHERE t3.prefix = :prefix ORDER BY t1.message_id DESC.
in C:\OpenServer\domains\ybugs.local\protected\models\ext\ExtMessage.php (55)
in C:\OpenServer\domains\ybugs.local\protected\controllers\LanguagesController.php (246)
in C:\OpenServer\domains\ybugs.local\protected\controllers\LanguagesController.php (276)
2015/03/13 18:26:53 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand failed to execute the SQL statement: CDbCommand failed to prepare the SQL statement: SQLSTATE[HY000]: General error: 1 no such column: t1.id. The SQL statement executed was: SELECT t1.id,t2.text,t1.translation, t2.id AS message_id
            FROM message_trl t1
            JOIN message t2 ON t1.message_id = t2.id
            JOIN languages t3 ON t1.language_id = t3.id
          WHERE t3.prefix = :prefix ORDER BY t1.message_id DESC' in C:\OpenServer\domains\ybugs.local\framework\db\CDbCommand.php:543
Stack trace:
#0 C:\OpenServer\domains\ybugs.local\framework\db\CDbCommand.php(396): CDbCommand->queryInternal('fetchAll', Array, Array)
#1 C:\OpenServer\domains\ybugs.local\protected\models\ext\ExtMessage.php(55): CDbCommand->queryAll(true, Array)
#2 C:\OpenServer\domains\ybugs.local\protected\controllers\LanguagesController.php(246): ExtMessage->getMessagesArr('en', Array)
#3 C:\OpenServer\domains\ybugs.local\protected\controllers\LanguagesController.php(276): LanguagesController->actionListMes()
#4 [internal function]: LanguagesController->actionMessages(NULL, NULL)
#5 C:\OpenServer\domains\ybugs.local\framework\web\actions\CAction.php(108): ReflectionMethod->invokeArgs(Object(LanguagesController), Array)
#6 C:\OpenServer\domains\ybugs.local\framework\web\actions\CInlineAction.php(47): CAction->runWithParamsInternal(Object(LanguagesController), Object(ReflectionMethod), Array)
#7 C:\OpenServer\domains\ybugs.local\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#8 C:\OpenServer\domains\ybugs.local\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#9 C:\OpenServer\domains\ybugs.local\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#10 C:\OpenServer\domains\ybugs.local\framework\web\CWebApplication.php(282): CController->run('messages')
#11 C:\OpenServer\domains\ybugs.local\framework\web\CWebApplication.php(141): CWebApplication->runController('languages/messa...')
#12 C:\OpenServer\domains\ybugs.local\framework\base\CApplication.php(180): CWebApplication->processRequest()
#13 C:\OpenServer\domains\ybugs.local\index.php(13): CApplication->run()
#14 {main}
REQUEST_URI=/en/languages/messages
HTTP_REFERER=http://ybugs.local/en/languages
---
2015/03/13 18:28:04 [error] [system.db.CDbCommand] Error in preparing SQL: SELECT t1.id,t2.text,t1.translation, t2.id AS message_id
            FROM message_trl t1
            JOIN message t2 ON t1.message_id = t2.id
            JOIN languages t3 ON t1.language_id = t3.id
          WHERE t3.prefix = :prefix ORDER BY t1.message_id DESC
in C:\OpenServer\domains\ybugs.local\protected\models\ext\ExtMessage.php (55)
in C:\OpenServer\domains\ybugs.local\protected\controllers\LanguagesController.php (246)
in C:\OpenServer\domains\ybugs.local\protected\controllers\LanguagesController.php (276)
2015/03/13 18:28:04 [error] [system.db.CDbCommand] CDbCommand::fetchAll() failed: CDbCommand failed to prepare the SQL statement: SQLSTATE[HY000]: General error: 1 no such column: t1.id. The SQL statement executed was: SELECT t1.id,t2.text,t1.translation, t2.id AS message_id
            FROM message_trl t1
            JOIN message t2 ON t1.message_id = t2.id
            JOIN languages t3 ON t1.language_id = t3.id
          WHERE t3.prefix = :prefix ORDER BY t1.message_id DESC.
in C:\OpenServer\domains\ybugs.local\protected\models\ext\ExtMessage.php (55)
in C:\OpenServer\domains\ybugs.local\protected\controllers\LanguagesController.php (246)
in C:\OpenServer\domains\ybugs.local\protected\controllers\LanguagesController.php (276)
2015/03/13 18:28:04 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand failed to execute the SQL statement: CDbCommand failed to prepare the SQL statement: SQLSTATE[HY000]: General error: 1 no such column: t1.id. The SQL statement executed was: SELECT t1.id,t2.text,t1.translation, t2.id AS message_id
            FROM message_trl t1
            JOIN message t2 ON t1.message_id = t2.id
            JOIN languages t3 ON t1.language_id = t3.id
          WHERE t3.prefix = :prefix ORDER BY t1.message_id DESC' in C:\OpenServer\domains\ybugs.local\framework\db\CDbCommand.php:543
Stack trace:
#0 C:\OpenServer\domains\ybugs.local\framework\db\CDbCommand.php(396): CDbCommand->queryInternal('fetchAll', Array, Array)
#1 C:\OpenServer\domains\ybugs.local\protected\models\ext\ExtMessage.php(55): CDbCommand->queryAll(true, Array)
#2 C:\OpenServer\domains\ybugs.local\protected\controllers\LanguagesController.php(246): ExtMessage->getMessagesArr('en', Array)
#3 C:\OpenServer\domains\ybugs.local\protected\controllers\LanguagesController.php(276): LanguagesController->actionListMes()
#4 [internal function]: LanguagesController->actionMessages(NULL, NULL)
#5 C:\OpenServer\domains\ybugs.local\framework\web\actions\CAction.php(108): ReflectionMethod->invokeArgs(Object(LanguagesController), Array)
#6 C:\OpenServer\domains\ybugs.local\framework\web\actions\CInlineAction.php(47): CAction->runWithParamsInternal(Object(LanguagesController), Object(ReflectionMethod), Array)
#7 C:\OpenServer\domains\ybugs.local\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#8 C:\OpenServer\domains\ybugs.local\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#9 C:\OpenServer\domains\ybugs.local\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#10 C:\OpenServer\domains\ybugs.local\framework\web\CWebApplication.php(282): CController->run('messages')
#11 C:\OpenServer\domains\ybugs.local\framework\web\CWebApplication.php(141): CWebApplication->runController('languages/messa...')
#12 C:\OpenServer\domains\ybugs.local\framework\base\CApplication.php(180): CWebApplication->processRequest()
#13 C:\OpenServer\domains\ybugs.local\index.php(13): CApplication->run()
#14 {main}
REQUEST_URI=/en/languages/messages
HTTP_REFERER=http://ybugs.local/en/languages
---
2015/03/13 18:37:20 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "js/messages.js".' in C:\OpenServer\domains\ybugs.local\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\OpenServer\domains\ybugs.local\framework\web\CWebApplication.php(141): CWebApplication->runController('js/messages.js')
#1 C:\OpenServer\domains\ybugs.local\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 C:\OpenServer\domains\ybugs.local\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/js/messages.js
HTTP_REFERER=http://ybugs.local/en/languages/messages
---
2015/03/13 18:37:31 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "js/messages.js".' in C:\OpenServer\domains\ybugs.local\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\OpenServer\domains\ybugs.local\framework\web\CWebApplication.php(141): CWebApplication->runController('js/messages.js')
#1 C:\OpenServer\domains\ybugs.local\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 C:\OpenServer\domains\ybugs.local\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/js/messages.js
HTTP_REFERER=http://ybugs.local/en/languages/messages
---
2015/03/13 18:37:36 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "js/messages.js".' in C:\OpenServer\domains\ybugs.local\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\OpenServer\domains\ybugs.local\framework\web\CWebApplication.php(141): CWebApplication->runController('js/messages.js')
#1 C:\OpenServer\domains\ybugs.local\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 C:\OpenServer\domains\ybugs.local\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/js/messages.js
HTTP_REFERER=http://ybugs.local/en/languages/messages
---
2015/03/13 18:37:40 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "js/messages.js".' in C:\OpenServer\domains\ybugs.local\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\OpenServer\domains\ybugs.local\framework\web\CWebApplication.php(141): CWebApplication->runController('js/messages.js')
#1 C:\OpenServer\domains\ybugs.local\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 C:\OpenServer\domains\ybugs.local\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/js/messages.js
HTTP_REFERER=http://ybugs.local/en/languages/listMes/curr_lng/en/search/
---
2015/03/13 19:10:42 [error] [php] Missing argument 2 for Logs::log_action(), called in C:\OpenServer\domains\ybugs.local\protected\controllers\MainController.php on line 46 and defined (C:\OpenServer\domains\ybugs.local\protected\helpers\Logs.php:26)
Stack trace:
#0 C:\OpenServer\domains\ybugs.local\framework\web\CController.php(308): CInlineAction->runWithParams()
#1 C:\OpenServer\domains\ybugs.local\framework\web\CController.php(286): MainController->runAction()
#2 C:\OpenServer\domains\ybugs.local\framework\web\CController.php(265): MainController->runActionWithFilters()
#3 C:\OpenServer\domains\ybugs.local\framework\web\CWebApplication.php(282): MainController->run()
#4 C:\OpenServer\domains\ybugs.local\framework\web\CWebApplication.php(141): CWebApplication->runController()
#5 C:\OpenServer\domains\ybugs.local\framework\base\CApplication.php(180): CWebApplication->processRequest()
#6 C:\OpenServer\domains\ybugs.local\index.php(13): CWebApplication->run()
REQUEST_URI=/en/main/login
in C:\OpenServer\domains\ybugs.local\protected\helpers\Logs.php (26)
in C:\OpenServer\domains\ybugs.local\protected\controllers\MainController.php (46)
in C:\OpenServer\domains\ybugs.local\index.php (13)
2015/03/13 19:12:26 [error] [php] fopen(C:\OpenServer\domains\ybugs.locallogs/log.txt) [<a href='function.fopen'>function.fopen</a>]: failed to open stream: No such file or directory (C:\OpenServer\domains\ybugs.local\protected\helpers\Logs.php:28)
Stack trace:
#0 C:\OpenServer\domains\ybugs.local\framework\web\actions\CInlineAction.php(49): MainController->actionLogin()
#1 C:\OpenServer\domains\ybugs.local\framework\web\CController.php(308): CInlineAction->runWithParams()
#2 C:\OpenServer\domains\ybugs.local\framework\web\CController.php(286): MainController->runAction()
#3 C:\OpenServer\domains\ybugs.local\framework\web\CController.php(265): MainController->runActionWithFilters()
#4 C:\OpenServer\domains\ybugs.local\framework\web\CWebApplication.php(282): MainController->run()
#5 C:\OpenServer\domains\ybugs.local\framework\web\CWebApplication.php(141): CWebApplication->runController()
#6 C:\OpenServer\domains\ybugs.local\framework\base\CApplication.php(180): CWebApplication->processRequest()
#7 C:\OpenServer\domains\ybugs.local\index.php(13): CWebApplication->run()
REQUEST_URI=/en/main/login
in C:\OpenServer\domains\ybugs.local\protected\helpers\Logs.php (28)
in C:\OpenServer\domains\ybugs.local\protected\controllers\MainController.php (46)
in C:\OpenServer\domains\ybugs.local\index.php (13)
2015/03/13 19:14:50 [error] [php] fopen(C:\OpenServer\domains\ybugs.local/logs/log.txt) [<a href='function.fopen'>function.fopen</a>]: failed to open stream: No such file or directory (C:\OpenServer\domains\ybugs.local\protected\helpers\Logs.php:28)
Stack trace:
#0 C:\OpenServer\domains\ybugs.local\framework\web\actions\CInlineAction.php(49): MainController->actionLogin()
#1 C:\OpenServer\domains\ybugs.local\framework\web\CController.php(308): CInlineAction->runWithParams()
#2 C:\OpenServer\domains\ybugs.local\framework\web\CController.php(286): MainController->runAction()
#3 C:\OpenServer\domains\ybugs.local\framework\web\CController.php(265): MainController->runActionWithFilters()
#4 C:\OpenServer\domains\ybugs.local\framework\web\CWebApplication.php(282): MainController->run()
#5 C:\OpenServer\domains\ybugs.local\framework\web\CWebApplication.php(141): CWebApplication->runController()
#6 C:\OpenServer\domains\ybugs.local\framework\base\CApplication.php(180): CWebApplication->processRequest()
#7 C:\OpenServer\domains\ybugs.local\index.php(13): CWebApplication->run()
REQUEST_URI=/en/main/login
in C:\OpenServer\domains\ybugs.local\protected\helpers\Logs.php (28)
in C:\OpenServer\domains\ybugs.local\protected\controllers\MainController.php (46)
in C:\OpenServer\domains\ybugs.local\index.php (13)
2015/03/13 19:17:35 [error] [php] fopen(C:\OpenServer\domains\ybugs.local\logs\log.txt) [<a href='function.fopen'>function.fopen</a>]: failed to open stream: No such file or directory (C:\OpenServer\domains\ybugs.local\protected\helpers\Logs.php:28)
Stack trace:
#0 C:\OpenServer\domains\ybugs.local\framework\web\actions\CInlineAction.php(49): MainController->actionLogin()
#1 C:\OpenServer\domains\ybugs.local\framework\web\CController.php(308): CInlineAction->runWithParams()
#2 C:\OpenServer\domains\ybugs.local\framework\web\CController.php(286): MainController->runAction()
#3 C:\OpenServer\domains\ybugs.local\framework\web\CController.php(265): MainController->runActionWithFilters()
#4 C:\OpenServer\domains\ybugs.local\framework\web\CWebApplication.php(282): MainController->run()
#5 C:\OpenServer\domains\ybugs.local\framework\web\CWebApplication.php(141): CWebApplication->runController()
#6 C:\OpenServer\domains\ybugs.local\framework\base\CApplication.php(180): CWebApplication->processRequest()
#7 C:\OpenServer\domains\ybugs.local\index.php(13): CWebApplication->run()
REQUEST_URI=/en/main/login
in C:\OpenServer\domains\ybugs.local\protected\helpers\Logs.php (28)
in C:\OpenServer\domains\ybugs.local\protected\controllers\MainController.php (46)
in C:\OpenServer\domains\ybugs.local\index.php (13)
2015/03/13 19:36:37 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "getlog".' in C:\OpenServer\domains\ybugs.local\framework\web\CController.php:483
Stack trace:
#0 C:\OpenServer\domains\ybugs.local\framework\web\CController.php(270): CController->missingAction('getlog')
#1 C:\OpenServer\domains\ybugs.local\framework\web\CWebApplication.php(282): CController->run('getlog')
#2 C:\OpenServer\domains\ybugs.local\framework\web\CWebApplication.php(141): CWebApplication->runController('languages/getlo...')
#3 C:\OpenServer\domains\ybugs.local\framework\base\CApplication.php(180): CWebApplication->processRequest()
#4 C:\OpenServer\domains\ybugs.local\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/en/languages/getlog
---
2015/03/13 19:36:52 [error] [php] file() [<a href='function.file'>function.file</a>]: Filename cannot be empty (C:\OpenServer\domains\ybugs.local\protected\helpers\Logs.php:42)
Stack trace:
#0 C:\OpenServer\domains\ybugs.local\framework\web\actions\CInlineAction.php(49): LanguagesController->actiongetLogs()
#1 C:\OpenServer\domains\ybugs.local\framework\web\CController.php(308): CInlineAction->runWithParams()
#2 C:\OpenServer\domains\ybugs.local\framework\web\CController.php(286): LanguagesController->runAction()
#3 C:\OpenServer\domains\ybugs.local\framework\web\CController.php(265): LanguagesController->runActionWithFilters()
#4 C:\OpenServer\domains\ybugs.local\framework\web\CWebApplication.php(282): LanguagesController->run()
#5 C:\OpenServer\domains\ybugs.local\framework\web\CWebApplication.php(141): CWebApplication->runController()
#6 C:\OpenServer\domains\ybugs.local\framework\base\CApplication.php(180): CWebApplication->processRequest()
#7 C:\OpenServer\domains\ybugs.local\index.php(13): CWebApplication->run()
REQUEST_URI=/en/languages/getlogs
in C:\OpenServer\domains\ybugs.local\protected\helpers\Logs.php (42)
in C:\OpenServer\domains\ybugs.local\protected\controllers\LanguagesController.php (564)
in C:\OpenServer\domains\ybugs.local\index.php (13)
2015/03/13 19:39:40 [error] [php] file(C:\OpenServer\domains\ybugs.localDSlogsDSlog.txt) [<a href='function.file'>function.file</a>]: failed to open stream: No such file or directory (C:\OpenServer\domains\ybugs.local\protected\helpers\Logs.php:43)
Stack trace:
#0 C:\OpenServer\domains\ybugs.local\framework\web\actions\CInlineAction.php(49): LanguagesController->actiongetLogs()
#1 C:\OpenServer\domains\ybugs.local\framework\web\CController.php(308): CInlineAction->runWithParams()
#2 C:\OpenServer\domains\ybugs.local\framework\web\CController.php(286): LanguagesController->runAction()
#3 C:\OpenServer\domains\ybugs.local\framework\web\CController.php(265): LanguagesController->runActionWithFilters()
#4 C:\OpenServer\domains\ybugs.local\framework\web\CWebApplication.php(282): LanguagesController->run()
#5 C:\OpenServer\domains\ybugs.local\framework\web\CWebApplication.php(141): CWebApplication->runController()
#6 C:\OpenServer\domains\ybugs.local\framework\base\CApplication.php(180): CWebApplication->processRequest()
#7 C:\OpenServer\domains\ybugs.local\index.php(13): CWebApplication->run()
REQUEST_URI=/en/languages/getlogs
in C:\OpenServer\domains\ybugs.local\protected\helpers\Logs.php (43)
in C:\OpenServer\domains\ybugs.local\protected\controllers\LanguagesController.php (564)
in C:\OpenServer\domains\ybugs.local\index.php (13)
